.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Issue
+about: Technical issue related to the code in this repository
+
+---
+
+> 🚨 This repository is for vehicle-command and its codebase. If your issue is not directly related to the code or this project, please visit our [Fleet API Support Page](https://fleetapi-support.tesla.com) for assistance.
```

Comment:
I think the scope of "vehicle-command" and "this project" could be misinterpreted, especially since "the code OR this project" suggests the scope of "this project" is different than the scope of the code.

I'd suggest a single sentence, "If you are not requesting a change to the code in this repository, please instead visit...".